scripts/ecure.lic

```diff
@@ -699,11 +725,11 @@ elsif Script.current.vars.include?('group')
   done = false
   healtargets = Array.new
   action = proc { |server_string|
-    if /^<a exist="-\d+" noun="(?'pcName'.*)">(?:.*)<\/a> is the leader of your group\./ =~ server_string
+    if /^<a exist="-\d+" noun="(?'pcName'.*)">(?:.*)<\/a> you are leading\./ =~ server_string
       healtargets.push(pcName)
       # groupleader = pcName
       nil
-    elsif /^<a exist="-\d+" noun="(?'pcName'.*)">(?:.*)<\/a> is also a member of your group\./ =~ server_string
+    elsif /^<a exist="-\d+" noun="(?'pcName'.*)">(?:.*)<\/a> you are grouped with\./ =~ server_string
       healtargets.push(pcName)
       nil
     elsif /^<a exist="-\d+" noun="(?'pcName'.*)">(?:.*)<\/a> is following you\./ =~ server_string
```

Comment:
Really only needed to remove some lines and update the regex, stolen from the Bigshot update.
```ruby
if /You are (?:leading|grouped with) (.*)\./i =~ server_string
```
and
```ruby
$1.split(/,(?:\s+and)?| and /).map { |x| x.gsub(/who is \w+|\(\w+\)|<a exist="-\d+" noun="\w+">|<\/a>/, '').strip }
```
So the whole action proc part then becomes this:
```ruby
  action = proc { |server_string|
    if /You are (?:leading|grouped with) (.*)\./i =~ server_string
      healtargets = $1.split(/,(?:\s+and)?| and /).map { |x| x.gsub(/who is \w+|\(\w+\)|<a exist="-\d+" noun="\w+">|<\/a>/, '').strip }
      # groupleader = pcName
      nil
    elsif /Your group status is currently/ =~ server_string
      nil
    elsif /You are not currently in a group/ =~ server_string
      nil
    elsif /<prompt/ =~ server_string
      nil
    elsif /GROUP HELP/ =~ server_string
      done = true
      DownstreamHook.remove('Group_hook')
      nil
    else
      server_string
    end
  }
```